apps/marketing/src/components/sections/landing/mocks.tsx

```
@@ -0,0 +1,355 @@
+        </div>
+        <div className="muted" style={{ fontSize: 12.5, marginTop: 14, display: "flex", alignItems: "center", gap: 8 }}>
+          <Ico name="canvas" size="i14" />Custom HTML &amp; CSS, rendered in an isolated sandbox.
+        </div>
+      </div>
+    </>
+  );
+}
+
+function CodeMock() {
+  const L: ReactNode[] = [
+    <span className="cm">// config.json — real-time collaboration</span>,
```

ESLint

[error] Missing "key" prop for element in array

(react/jsx-key)